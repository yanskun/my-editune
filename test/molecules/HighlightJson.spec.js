import { createLocalVue, shallowMount } from "@vue/test-utils"
import ElementUI from "element-ui"
import Component from "~/components/molecules/HighlightJson"

const localVue = createLocalVue()
localVue.use(ElementUI)

describe("HighlightJson", () => {
  test("snapshot", () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        json: {
          string: "value",
          number: 0,
          boolean: false
        }
      },
      localVue
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
