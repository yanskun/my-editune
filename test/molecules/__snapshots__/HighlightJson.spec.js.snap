// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightJson snapshot 1`] = `
<pre>
  {
	
  <span
    class="hljs-attr"
  >
    "string"
  </span>
  : 
  <span
    class="hljs-string"
  >
    "value"
  </span>
  ,
	
  <span
    class="hljs-attr"
  >
    "number"
  </span>
  : 
  <span
    class="hljs-number"
  >
    0
  </span>
  ,
	
  <span
    class="hljs-attr"
  >
    "boolean"
  </span>
  : 
  <span
    class="hljs-literal"
  >
    false
  </span>
  
}
</pre>
`;
